src/shared/bg_teamprogress.cpp

```diff
@@ -272,8 +272,10 @@ void BG_ImportUnlockablesFromMask( int team, int mask )
 	lastMask = mask;
 	lastTeam = (team_t) team;
 
+#ifdef BUILD_CGAME
 	// no status change yet
 	memset( statusChanges, 0, sizeof( statusChanges ) );
```

Comment:
Also move the declaration of statusChanges inside the ifdef?

Reply:
Done